dashboard/src/utils/apiConstants.js

```diff
@@ -1,2 +1 @@
-export const LOCAL_BASE_URL = "/";
-
+export const LOCAL_BASE_URL = "http://10.1.170.201/";
```

Comment:
Is there a reason for this hard-coded IP address?

Reply:
Yes, we need to move quickly (not necessarily in this PR) to automatic configuration through the connected server, e.g. with a `fetch('/api/v1/endpoints')` early in the master page load (`index.html` or `App.js` infrastructure). There's an epic (currently with one story) reflecting this in Jira: [PBENCH-65](https://issues.redhat.com/browse/PBENCH-65).

Reply:
Note there's a draft PR to begin moving towards fixing this, #2762